libcloud/compute/drivers/ovh.py

```diff
@@ -44,7 +44,8 @@ class OvhNodeDriver(NodeDriver):
     VOLUME_STATE_MAP = OpenStackNodeDriver.VOLUME_STATE_MAP
     SNAPSHOT_STATE_MAP = OpenStackNodeDriver.SNAPSHOT_STATE_MAP
 
-    def __init__(self, key, secret, ex_project_id, ex_consumer_key=None):
+    def __init__(self, key, secret, ex_project_id, ex_consumer_key=None,
+                 ex_datacenter=None):
```

Comment:
For consistency and compatibility with other drivers which, you should use existing ``region`` argument - AWS and many other drivers which support multiple regions / datacenters support that argument.

Reply:
That makes sense. Update inbound. (I'd just kept the existing names originally.)